### PR TITLE
STYLE:PEP8 for test_imaffine

### PR DESCRIPTION
**Pull request by kennyhothedev**
pep 8 for dipy/align/tests/test_imaffine.py #864

### CODE REVIEW

File: dipy/align/tests/test_imaffine.py [hunk 1]
```diff
@@ -117,28 +117,28 @@ def test_transform_geometric_centers_3d():
...
                     c_moving = tuple(c_moving)
                     c_moving = moving_grid2world.dot(c_moving+(1,))[:3]
-                    expected = np.eye(4);
+                    expected = np.eye(4)
                     expected[:3, 3] = c_moving - c_static
 
                     # Implementation under test
-                    actual = imaffine.transform_geometric_centers(static,
-                        static_grid2world, moving, moving_grid2world)
+                    actual = imaffine.transform_geometric_centers
```
Comment (omarocegueda) [inline dipy/align/tests/test_imaffine.py]: Here, function `imaffine.transform_geometric_centers` is not being called. Instead, `actual` will be assigned a reference to the function, that's why Travis failing. To call the function, at least the opening parenthesis `(` must appear before breaking the line.

File: dipy/align/tests/test_imaffine.py [hunk 2]
```diff
@@ -164,12 +164,12 @@ def test_transform_origins_3d():
...
                     c_static = static_grid2world[:3, 3]
                     c_moving = moving_grid2world[:3, 3]
-                    expected = np.eye(4);
+                    expected = np.eye(4)
                     expected[:3, 3] = c_moving - c_static
 
                     # Implementation under test
-                    actual = imaffine.transform_origins(static, static_grid2world,
-                                                    moving, moving_grid2world)
+                    actual = imaffine.transform_origins
```
Comment (omarocegueda) [inline dipy/align/tests/test_imaffine.py]: Same issue as above, function `imaffine.transform_origins` is not being called

**Comment by omarocegueda** [comment]
Hi @hokatvcu!, =)
thank you very much for your contribution! in addition to my comments above, I just checked with Spyder, and it still reports several PEP8 issues. By any chance are you using an editor to check PEP8 (Spyder is a very good option)?
Also, to verify that nothing got broken after your changes, you can open a shell in the tests directory (in this case: `dipy/dipy/align/tests`), and run the tests, for example in this case:
```nosetests -v --stop test_imaffine.py```
Thanks a lot!

**Comment by hokatvcu** [comment]
I'm using atom. Thanks for the advice!

**Comment by riddhishb** [comment]
I want to work on this issue

**Comment by arokem** [comment]
It seems that this PR was abandoned. Closing in favor of #964

**Pull request closed without merging**